src/subcommand/list.rs

```diff
@@ -2,22 +2,36 @@ use super::*;
 
 #[derive(Debug, Parser)]
 pub(crate) struct List {
-  outpoint: OutPoint,
+  #[clap(long, short, help = "Use extended output format")]
```

Comment:
I think the extended output format code snuck in here.

Reply:
Ups, yes. I must have messed something up after I had merged those changes into my master.